MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -936,22 +930,28 @@ private void BuildStreamVideoItem(
             }
 
             // Prefer matching audio codecs, could do better here
-            var audioCodecs = ContainerProfile.SplitValue(audioCodec);
+            List<string> audioCodecs = [.. ContainerHelper.Split(audioCodec)];
```

Comment:
I still prefer ToArray and ToList in these cases, way clearer IMHO
(not just here, but all over the PR BTW)

Reply:
Agreed, I reverted it.